auth-web/src/router.ts

```diff
@@ -96,7 +96,7 @@ export function getRoutes (): RouteConfig[] {
     },
     { path: '/setup-account', name: 'setupaccount', component: AccountSetupView, props: true, meta: { requiresAuth: true, requiresProfile: true } },
     { path: '/setup-account-success', name: 'setup-account-success', component: AccountCreationSuccessView, meta: { requiresAuth: true, requiresProfile: true } },
-    { path: '/userprofile/:token?', name: 'userprofile', component: UserProfileView, props: true, meta: { requiresAuth: true } },
+    { path: '/userprofile/:token?', name: 'userprofile', component: UserProfileView, props: true, meta: { requiresAuth: true, requiresProfile: true } },
```

Comment:
User Profile needs UserProfile ?

Reply:
we have to rename the flag. now requiresProfile is more of 'requires terms of service'...so user profile now needs terms of service...

Reply:
ohh ok